taskweaver/planner/planner.py

```diff
@@ -195,31 +197,53 @@ def check_post_validity(post: Post):
             assert post.attachment_list[1].type == "plan", "attachment type is not plan"
             assert post.attachment_list[2].type == "current_plan_step", "attachment type is not current_plan_step"
 
-        llm_output = self.llm_api.chat_completion(chat_history, use_backup_engine=use_back_up_engine)["content"]
-        try:
-            response_post = self.planner_post_translator.raw_text_to_post(
-                llm_output=llm_output,
-                send_from="Planner",
-                event_handler=event_handler,
-                validation_func=check_post_validity,
-            )
-            if response_post.send_to == "User":
-                event_handler("final_reply_message", response_post.message)
-        except (JSONDecodeError, AssertionError) as e:
-            self.logger.error(f"Failed to parse LLM output due to {str(e)}")
+        if self.config.skip_planning and rounds[-1].post_list[-1].send_from == "User":
             response_post = Post.create(
-                message=f"Failed to parse Planner output due to {str(e)}."
-                f"The output format should follow the below format:"
-                f"{self.prompt_data['planner_response_schema']}"
-                "Please try to regenerate the output.",
-                send_to="Planner",
+                message="Please reply to the user's request: " + rounds[-1].post_list[-1].message,
```

Comment:
May change "Please reply to the user's request:" to "Please process this request:" because CI has no idea of what this "user" is.

Reply:
Revised, also extracted a json file to load the default plan.